src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -444,9 +447,11 @@ def work_for_vts_rsc(vts, ctx):
         classpath_directory_digests = []
         classpath_product = self.context.products.get_data('rsc_mixed_compile_classpath')
         classpath_entries = classpath_product.get_classpath_entries_for_targets(dependencies_for_target)
+
+        hermetic = self.execution_strategy == self.ExecutionStrategy.hermetic
         for _conf, classpath_entry in classpath_entries:
           classpath_paths.append(fast_relpath(classpath_entry.path, get_buildroot()))
-          if self.execution_strategy == self.ExecutionStrategy.hermetic and not classpath_entry.directory_digest:
+          if hermetic and not classpath_entry.directory_digest:
```

Comment:
`not classpath_entry.directory_digest`
is this necessary? I think we've fixed all the digest issues.

Reply:
Not sure. Let's leave it in for now since this was the previous implementation.